src/editors/containers/VideoUploadEditor/hooks.js

```diff
@@ -54,22 +49,17 @@ export const onVideoUpload = () => {
 
 const getFileExtension = (filename) => filename.slice(Math.abs(filename.lastIndexOf('.') - 1) + 2);
 
-export const fileValidator = (setLoading, setErrorMessage, uploadVideo) => (file) => {
-  const supportedFormats = Object.keys(extToMime);
+export const fileValidator = (setLoading, uploadVideo) => (fileData) => {
+  const file = fileData.getAll('file')[0];
   const ext = getFileExtension(file.name);
   const type = extToMime[ext] || '';
   const newFile = new File([file], file.name, { type });
 
-  if (supportedFormats.includes(ext)) {
-    uploadVideo({
-      supportedFiles: [newFile],
-      setLoadSpinner: setLoading,
-      postUploadRedirect: onVideoUpload(),
-    });
-  } else {
-    const errorMsg = 'Video must be an MP4 or MOV file';
-    setErrorMessage(errorMsg);
-  }
+  uploadVideo({
+    supportedFiles: [newFile],
+    setLoadSpinner: setLoading,
+    postUploadRedirect: onVideoUpload(),
+  });
 };
```

Comment:
@0x29a I think the dropzone component already does validation, so it might be OK to skip it here. 

That said, I don't understand what this function is doing. Are browsers not filling in the mime type? I tried a couple of files and the type is already set correctly, so you can just use the original file instead of creating a new one. I agree that it's no longer a validator, so the name should change. I think removing that function can calling uploadVideo directly would be better.

Reply:
@0x29a @xitij2000  you are write we no longer need this function since Dropzone can handle this.